fiftyone/server/routes/values.py

```diff
@@ -36,13 +36,20 @@ async def post(self, request: Request, data: dict):
 
         sort_by = "count" if count else "_id"
 
+        # escape special characters, add support for wilcard pattern matching
+        regex_safe_search = (
+            re.escape(search).replace(r"\*", ".*").replace(r"\?", ".")
```

Comment:
I think this replacement doesn't work if a user starts with valid regex. 

https://user-images.githubusercontent.com/30936736/223288932-1ac61c29-ef1e-40d1-bb7b-751dcc61b5e0.mov

Reply:
In the frontend, we're supporting wildcard pattern matching, not regex matching. With wildcard pattern matching, the text you specified `.*kacey.*` would only match strings like `.ABCDkacey.ABCDEF`.